docs/src/main/asciidoc/security-enabling-basic-auth-tutorial.adoc

```diff
@@ -0,0 +1,24 @@
+[id="security-enabling-basic-auth-tutorial"]
+= Enabling Basic Authentication
```

Comment:
```suggestion
= Enable Basic Authentication
```

Reply:
Hello Erin. We use action-oriented headings for procedure-like content.

Reply:
Enable is still action oriented.

There is an interesting balance to strike for descriptiveness in document titles and the preamble (descriptive and searchable, without being too long so it can be included nicely in generated pages that list all the things).

The title of a tutorial should capture the end state (rather than the path to get there). I tried to give an example for tutorial titles, i.e. they should complete a sentence like: "You will create an application that [uses x extension to] …​" or... "In this tutorial you will..."